src/app/roles.tsx

```diff
@@ -22,14 +22,20 @@ const pathsShownToAllAuthenticatedUsers = [
     "/set-password",
 ] as const;
 
+const pathsOnlyShownToManager = ["/lists"] as const;
+
 const pathsOnlyShownToAdmin = ["/admin", "/lists"] as const;
 
 const getShownPagesByRole = (role: UserRole | null): readonly string[] => {
     switch (role) {
         case "admin":
             return [...pathsShownToAllAuthenticatedUsers, ...pathsOnlyShownToAdmin];
-        case "caller":
+        case "volunteer":
             return pathsShownToAllAuthenticatedUsers;
+        case "manager":
+            return [...pathsShownToAllAuthenticatedUsers, ...pathsOnlyShownToManager];
+        case "staff":
+            return [...pathsShownToAllAuthenticatedUsers, ...pathsOnlyShownToManager];
```

Comment:
since we have a very clear 1-dimentional structure of permissions, why don't we "incrementally add paths as we go up the roles" to avoid repetition? i.e.
```ts
const pathsShownToStaffAndAbove = ["/lists"] as const;
const const pathsOnlyShownToAdmin = ["/admin"] as const;

...

    case "admin:
        return [...pathsShownToAllAuthenticatedUsers, ...pathsShownToStaffAndAbove, ...pathsOnlyShownToAdmin];

Reply:
Sure, I wasn't sure whether you would prefer this or what I did but will do